.github/actions/functional-test-cucumber/action.yml

```
@@ -0,0 +1,19 @@
+name: Functional Tests
+description: Run Functional Tests
+runs:
+  using: "composite"
+  steps:
+    - run: npm install --silent --unsafe-perm
```

Why did you use --unsafe-perm? It seems to me that runners are not containers
```suggestion
    - run: npm install --silent
```